cmake/setup_cross_compile.sh

```diff
@@ -0,0 +1,4 @@
+# Set the following variables as per tool installation on your system
+export CROSS_COMPILER_PATH=$HOME/GIT_DIR/REPO/gcc-arm-9.2-2019.12-x86_64-aarch64-none-linux-gnu
```

Comment:
@s-jain1 Should this be the new GCC 11.3 compiler?

Reply:
Updated the PR with GCC 11.3 compiler